odbc/src/api/bind_col_tests.rs

```diff
@@ -0,0 +1,319 @@
+mod unit {
```

Comment:
A lot of my tests use repeated code; however, when I tried to refactor and condense, I was getting some lifetime related errors, so I decided not to take the time to condense everything yet. However, I can spend more time on this and condense it if that's preferable.